src/api/controllers/articleController.js

```diff
@@ -92,17 +92,28 @@ export default class ArticleController {
     const articles = await Article.findAndCountAll({
       offset,
       limit,
-      attributes: ['id', 'title', 'description', 'body', 'slug', 'coverImage', 'tagList', 'createdAt', 'updatedAt'],
+      attributes: [
```

Comment:
@Ramadhan0 why is this showing up in your work?

Reply:
this is vscode, i didn't add or remove anything @MemunaHaruna